backend/src/apiserver/metadata/metadata_store.go

```diff
@@ -0,0 +1,119 @@
+package metadata
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	argoWorkflow "github.com/argoproj/argo/pkg/apis/workflow/v1alpha1"
+	"github.com/golang/protobuf/jsonpb"
+	"github.com/golang/protobuf/proto"
+	"github.com/kubeflow/pipelines/backend/src/common/util"
+
+	"ml_metadata/metadata_store/mlmetadata"
+	mlpb "ml_metadata/proto/metadata_store_go_proto"
+)
+
+// Store ...
+type Store struct {
+	mlmdStore *mlmetadata.Store
+}
+
+// NewStore ...
+func NewStore(mlmdStore *mlmetadata.Store) *Store {
+	return &Store{mlmdStore: mlmdStore}
+}
+
+// RecordOutputArtifacts ...
+func (s *Store) RecordOutputArtifacts(runID string, storedManifest, currentManifest string) error {
```

Comment:
could you make it consistent 
runID, storedManifest, currentManifest string

Reply:
also maybe name storedManifest -> storedWorkflowString? currentManifest -> currentWorkflowString